LICENSE

```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016 Daniel Cohen Gindi & Philipp Jahoda
```

Comment:
@danielgindi is this intended to use a template or forgot to change?
BTW, should it be 2017 or just keep the first year?

Reply:
It was missed. We can update the year later